### PR TITLE
Add Translation Program mission and vision section

### PR DESCRIPTION
**Pull request by lukassim**
## Description

Added a section detailing the Translation Program mission and vision.
Added a section linking to the blog posts on the EF blog.

Preview deploy: https://deploy-preview-4046--ethereumorg.netlify.app/en/contributing/translation-program/#mission-and-vision

## Related Issue

#4036

### CODE REVIEW

File: src/content/contributing/translation-program/index.md [hunk 1]
```diff
@@ -106,6 +106,39 @@ Thank you for your participation in the ethereum.org Translation Program!
 
 - [Content versions](/contributing/translation-program/content-versions/) _– Which pages are included in each content version of ethereum.org_
 
+## Translation Program mission and vision {#mission-and-vision}
+
+The Ethereum community is truly global and all-inclusive. However, as over 3/4 of the world’s population do not speak English, we want to provide them with content in their native language, in order to be the portal to Ethereum for the global community.
```
Comment (samajammin) [inline src/content/contributing/translation-program/index.md:111]:
```suggestion
The Ethereum community strives to be truly global and all-inclusive. Given that over 3/4 of the world’s population do not speak English, we want to provide them with content in their native language, in order for ethereum.org to be the portal to Ethereum for the global community.
```

File: src/content/contributing/translation-program/index.md [hunk 2]
```diff
@@ -106,6 +106,39 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+
+The aim of the ethereum.org Translation Program is to make Ethereum accessible to everyone, regardless of their nationality and language.
+
+To achieve this, our community of volunteer translators is working on translating the website and other Ethereum content into as many languages as possible.
```
Comment (samajammin) [inline src/content/contributing/translation-program/index.md:115]: I think this sentence could be brought into the paragraph above.

File: src/content/contributing/translation-program/index.md [hunk 3]
```diff
@@ -106,6 +106,39 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+
+**Our mission**
+
+- By providing translated versions of the website, visitors from all over the world can learn about Ethereum in their native language.
```
Comment (samajammin) [inline src/content/contributing/translation-program/index.md:119]:
```suggestion
- Provide translated versions of the website to enable visitors from all over the world to learn about Ethereum in their native language.
```

File: src/content/contributing/translation-program/index.md [hunk 4]
```diff
@@ -106,6 +106,39 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+    - This helps to bridge language gaps and provides visitors with more inclusive and more generally understandable content.
```
Comment (samajammin) [inline src/content/contributing/translation-program/index.md:120]:
```suggestion
    - Bridge language gaps and provide visitors with more inclusive and more generally understandable content.
```

File: src/content/contributing/translation-program/index.md [hunk 5]
```diff
@@ -106,6 +106,39 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+    - In addition, this facilitates the onboarding of more members to the global Ethereum community.
```
Comment (samajammin) [inline src/content/contributing/translation-program/index.md:121]:
```suggestion
    - Facilitate the onboarding of more members to the global Ethereum community.
```

File: src/content/contributing/translation-program/index.md [hunk 6]
```diff
@@ -106,6 +106,39 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+- Translated content about Ethereum allows for easier and more inclusive sharing of information and knowledge.
```
Comment (samajammin) [inline src/content/contributing/translation-program/index.md:122]:
```suggestion
- Allow for easier and more inclusive sharing of Ethereum information and knowledge.
```

File: src/content/contributing/translation-program/index.md [hunk 7]
```diff
@@ -106,6 +106,39 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+- By opening up the Translation Program to voluntary contributions, community members from all over the world have a chance to contribute to Ethereum and make their mark on the ecosystem.
```
Comment (samajammin) [inline src/content/contributing/translation-program/index.md:123]:
```suggestion
- Open up the Translation Program to voluntary contributions to empower community members from all over the world to have a chance to contribute to Ethereum and make their mark on the ecosystem.
```

File: src/content/contributing/translation-program/index.md [hunk 8]
```diff
@@ -106,6 +106,39 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+- The Translation Program also allows us to identify and connect with numerous supporters and contributors, looking to get involved with Ethereum.
```
Comment (samajammin) [inline src/content/contributing/translation-program/index.md:124]:
```suggestion
- Identify, connect with, and engage numerous supporters and contributors looking to get involved with Ethereum.
```

File: src/content/contributing/translation-program/index.md [hunk 9]
```diff
@@ -106,6 +106,39 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+
+**Our vision**
+
+- To translate essential content for members of the Ethereum community from as many countries and parts of the world as possible.
```
Comment (samajammin) [inline src/content/contributing/translation-program/index.md:128]:
```suggestion
- Translate essential content for members of the Ethereum community from as many countries and parts of the world as possible.
```

File: src/content/contributing/translation-program/index.md [hunk 10]
```diff
@@ -106,6 +106,39 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+- To support the sharing of knowledge across languages with the aim of creating a better informed and educated Ethereum community.
```
Comment (samajammin) [inline src/content/contributing/translation-program/index.md:129]:
```suggestion
- Support knowledge sharing across languages with the aim of creating a better informed and educated Ethereum community.
```

File: src/content/contributing/translation-program/index.md [hunk 11]
```diff
@@ -106,6 +106,39 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+- To increase Ethereum's inclusivity and accessibility by providing translated content and removing the language barriers that prevent non-English speakers from joining the ecosystem.
```
Comment (samajammin) [inline src/content/contributing/translation-program/index.md:130]:
```suggestion
- Increase Ethereum's inclusivity and accessibility by providing translated content and removing the language barriers that prevent non-English speakers from joining the ecosystem.
```

File: src/content/contributing/translation-program/index.md [hunk 12]
```diff
@@ -106,6 +106,39 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+
+**Ethereum Foundation blog**
```
Comment (samajammin) [inline src/content/contributing/translation-program/index.md:132]:
```suggestion
**Latest updates**
```

File: src/content/contributing/translation-program/index.md [hunk 13]
```diff
@@ -106,6 +106,39 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+
+To keep in touch with the latest updates about the Translation Program, you can follow the [Ethereum Foundation blog](https://blog.ethereum.org/).
```
Comment (samajammin) [inline src/content/contributing/translation-program/index.md:134]:
```suggestion
To keep in touch with the latest Translation Program progress, you can follow the [Ethereum Foundation blog](https://blog.ethereum.org/).
```

File: src/content/contributing/translation-program/index.md [hunk 14]
```diff
@@ -106,6 +106,39 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+
+Our older posts are available below:
```
Comment (samajammin) [inline src/content/contributing/translation-program/index.md:136]:
```suggestion
Our most recent posts are available below:
```

File: src/content/contributing/translation-program/index.md [hunk 15]
```diff
@@ -106,6 +106,39 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+
+- [Translation Program announcement](https://blog.ethereum.org/2019/08/20/translating-ethereum-for-our-global-community/)
```
Comment (samajammin) [inline src/content/contributing/translation-program/index.md:138]:
```suggestion
- [August 2019 Translation Program launch](https://blog.ethereum.org/2019/08/20/translating-ethereum-for-our-global-community/)
```

File: src/content/contributing/translation-program/index.md [hunk 16]
```diff
@@ -106,6 +106,39 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+- [July 2020 post](https://blog.ethereum.org/2020/07/29/ethdotorg-translation-milestone/)
```
Comment (samajammin) [inline src/content/contributing/translation-program/index.md:139]:
```suggestion
- [July 2020 milestones update](https://blog.ethereum.org/2020/07/29/ethdotorg-translation-milestone/)
```

File: src/content/contributing/translation-program/index.md [hunk 17]
```diff
@@ -106,6 +106,39 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+- [December 2020 post](https://blog.ethereum.org/2020/12/21/translation-program-milestones-updates-20/)
```
Comment (samajammin) [inline src/content/contributing/translation-program/index.md:140]:
```suggestion
- [December 2020 milestones update](https://blog.ethereum.org/2020/12/21/translation-program-milestones-updates-20/)
```

**Review by samajammin — COMMENTED** [review]
Looks great! Made a bunch of minor suggestions, mostly updating to use a consistent active voice in the bullet points.

File: src/content/contributing/translation-program/index.md [hunk 18]
```diff
@@ -106,6 +106,39 @@ Thank you for your participation in the ethereum.org Translation Program!
 
 - [Content versions](/contributing/translation-program/content-versions/) _– Which pages are included in each content version of ethereum.org_
 
+## Translation Program mission and vision {#mission-and-vision}
```
Comment (samajammin) [inline src/content/contributing/translation-program/index.md:109]:
```suggestion
## Mission and vision {#mission-and-vision}
```

Comment (samajammin) [reply src/content/contributing/translation-program/index.md:109]: Should this be the top section on the page (i.e. in front of "How to translate" & "Resources")?

File: src/content/contributing/translation-program/index.md (hunk 7 above)
Comment (wackerow) [inline src/content/contributing/translation-program/index.md:123]:
```suggestion
- Work with voluntary contributors and community members from all over the world to provide a chance for anyone to contribute to Ethereum and make their mark on the ecosystem.
```

File: src/content/contributing/translation-program/index.md (hunk 8 above)
Comment (wackerow) [inline src/content/contributing/translation-program/index.md:124]:
```suggestion
- Identify and connect with supporters and contributors looking to get involved with Ethereum and help provide guidance.
```

**Review by wackerow — CHANGES_REQUESTED** [review]
@lukassim Great start here! Made a few comments more related to wording/tone of what we're labeling "our mission," as it seems like it should be a bit more active.

Otherwise this looks great. Was trying to think of a way to squeeze in how Ethereum is something that stands to benefit these groups the most in many ways, and the option of leaving everything in English would exclude the majority of people who need this most, while enabling parts of society that are already fairly well off. Feel free to ignore or run with that thought =)

Nice job!

File: src/content/contributing/translation-program/index.md [hunk 19]
```diff
@@ -106,6 +106,36 @@ Thank you for your participation in the ethereum.org Translation Program!
 
 - [Content versions](/contributing/translation-program/content-versions/) _– Which pages are included in each content version of ethereum.org_
 
+## Mission and vision {#mission-and-vision}
+
+The Ethereum community strives to be truly global and all-inclusive. Given that over 3/4 of the world’s population do not speak English, we want to provide them with content in their native language, in order for ethereum.org to be the portal to Ethereum for the global community.
+
+The aim of the ethereum.org Translation Program is to make Ethereum accessible to everyone, regardless of their nationality and language, by translating the website and other Ethereum content into as many languages as possible.
```
Comment (minimalsm) [inline src/content/contributing/translation-program/index.md:113]:
```suggestion
The ethereum.org Translation Program aims to make Ethereum accessible to everyone by translating ethereum.org and other Ethereum content into as many languages as possible.
```

Comment (minimalsm) [reply src/content/contributing/translation-program/index.md:113]: >  regardless of their nationality and language

This is already implied

File: src/content/contributing/translation-program/index.md [hunk 20]
```diff
@@ -106,6 +106,36 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+
+**Our mission**
+
+- Provide translated versions of the website to empower visitors from all over the world to learn about Ethereum in their native language.
```
Comment (minimalsm) [inline src/content/contributing/translation-program/index.md:117]:
```suggestion
- Provide translated versions of the website to empower visitors worldwide to learn about Ethereum in their native language
```

File: src/content/contributing/translation-program/index.md [hunk 21]
```diff
@@ -106,6 +106,36 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+- Facilitate the onboarding of those who stand to benefit the most from Ethereum
```
Comment (minimalsm) [inline src/content/contributing/translation-program/index.md:118]: > Facilitate the onboarding of those who stand to benefit the most from Ethereum

This seems a bit hand-wavy to me. Who benefits the most?

File: src/content/contributing/translation-program/index.md [hunk 22]
```diff
@@ -106,6 +106,36 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+- Allow for easier and more inclusive sharing of Ethereum information and knowledge.
```
Comment (minimalsm) [inline src/content/contributing/translation-program/index.md:119]:
```suggestion
- Allow for more accessible and more inclusive sharing of Ethereum information and knowledge
```

File: src/content/contributing/translation-program/index.md [hunk 23]
```diff
@@ -106,6 +106,36 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+- Open up the Translation Program to voluntary contributions to empower community members from all over the world to have a chance to contribute to Ethereum and make their mark on the ecosystem.
```
Comment (minimalsm) [inline src/content/contributing/translation-program/index.md:120]:
```suggestion
- Empower community members to contribute translations to Ethereum and make their mark on the ecosystem
```

> Open up the Translation Program to voluntary contribution

I think this is pretty well implied already

File: src/content/contributing/translation-program/index.md [hunk 24]
```diff
@@ -106,6 +106,36 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+- Identify and connect with supporters and contributors looking to get involved with Ethereum and help provide guidance.
```
Comment (minimalsm) [inline src/content/contributing/translation-program/index.md:121]:
```suggestion
- Identify, connect with, and provide guidance to passionate contributors looking to get involved in the ecosystem
```

File: src/content/contributing/translation-program/index.md [hunk 25]
```diff
@@ -106,6 +106,36 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+
+**Our vision**
+
+- Translate essential content for members of the Ethereum community from as many countries and parts of the world as possible.
+- Support knowledge sharing across languages with the aim of creating a better informed and educated Ethereum community.
+- Increase Ethereum's inclusivity and accessibility by providing translated content and removing the language barriers that prevent non-English speakers from joining the ecosystem.
```
Comment (minimalsm) [inline src/content/contributing/translation-program/index.md:127]:
```suggestion
- Increase Ethereum's inclusivity and accessibility by removing the language barriers preventing non-English speakers from joining the ecosystem
```

File: src/content/contributing/translation-program/index.md [hunk 26]
```diff
@@ -106,6 +106,36 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+- Provide translated versions of the website to empower visitors from all over the world to learn about Ethereum in their native language.
+- Facilitate the onboarding of those who stand to benefit the most from Ethereum
+- Allow for easier and more inclusive sharing of Ethereum information and knowledge.
+- Open up the Translation Program to voluntary contributions to empower community members from all over the world to have a chance to contribute to Ethereum and make their mark on the ecosystem.
+- Identify and connect with supporters and contributors looking to get involved with Ethereum and help provide guidance.
+
+**Our vision**
+
+- Translate essential content for members of the Ethereum community from as many countries and parts of the world as possible.
+- Support knowledge sharing across languages with the aim of creating a better informed and educated Ethereum community.
```
Comment (minimalsm) [inline src/content/contributing/translation-program/index.md:126]:
```suggestion
- Support knowledge sharing across languages to create a better informed and educated Ethereum community
```

File: src/content/contributing/translation-program/index.md [hunk 27]
```diff
@@ -106,6 +106,36 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+
+- Provide translated versions of the website to empower visitors from all over the world to learn about Ethereum in their native language.
+- Facilitate the onboarding of those who stand to benefit the most from Ethereum
+- Allow for easier and more inclusive sharing of Ethereum information and knowledge.
+- Open up the Translation Program to voluntary contributions to empower community members from all over the world to have a chance to contribute to Ethereum and make their mark on the ecosystem.
+- Identify and connect with supporters and contributors looking to get involved with Ethereum and help provide guidance.
+
+**Our vision**
+
+- Translate essential content for members of the Ethereum community from as many countries and parts of the world as possible.
```
Comment (minimalsm) [inline src/content/contributing/translation-program/index.md:125]: > essential content

How do we define essential content?

File: src/content/contributing/translation-program/index.md [hunk 28]
```diff
@@ -106,6 +106,36 @@ Thank you for your participation in the ethereum.org Translation Program!
...
+
+**Latest updates**
+
+To keep in touch with the latest Translation Program progress, you can follow the [Ethereum Foundation blog](https://blog.ethereum.org/).
+
+Our most recent posts are available below:
+
+- [August 2019 Translation Program launch](https://blog.ethereum.org/2019/08/20/translating-ethereum-for-our-global-community/)
+- [July 2020 milestones update](https://blog.ethereum.org/2020/07/29/ethdotorg-translation-milestone/)
+- [December 2020 milestones update](https://blog.ethereum.org/2020/12/21/translation-program-milestones-updates-20/)
```
Comment (minimalsm) [inline src/content/contributing/translation-program/index.md:137]:
```suggestion
To keep up-to-date with the latest Translation Program progress, you can follow the [Ethereum Foundation blog](https://blog.ethereum.org/):

- [December 2020 milestones update](https://blog.ethereum.org/2020/12/21/translation-program-milestones-updates-20/)
- [July 2020 milestones update](https://blog.ethereum.org/2020/07/29/ethdotorg-translation-milestone/)
- [August 2019 Translation Program launch](https://blog.ethereum.org/2019/08/20/translating-ethereum-for-our-global-community/)
```

IMO the updates should be in reverse chronological order (we already do this elsewhere)

Screenshot 2021-09-28 at 11 21 03

> Our most recent posts are available below:

We can probably drop this, it's pretty implied / already has the law of proximity working for it.

**Review by minimalsm — CHANGES_REQUESTED** [review]
@lukassim looking awesome 🙌❤️ Made a few suggestions but feel free to drop anything you disagree with.

**Review by wackerow — CHANGES_REQUESTED** [review]
This looks great @lukassim! After taking a look at it as the whole page, my vote would be to have this higher up on the page. 

Would personally suggest moving the Mission and Vision section up just above `#how-to-translate`, but splitting of the `Latest updates` as it's own Section and leaving it where it is. 

This way people are given a quick intro, then a passion-driven mission statement of _why_ this matters, then on to _how_ to be a part of it.

Loving it =) Nice job

File: src/content/contributing/translation-program/index.md (hunk 27 above)
Comment (minimalsm) [reply src/content/contributing/translation-program/index.md]: On second read I'm okay with this being vague 👍

**Review by wackerow — APPROVED** [review]
💪🏻 🎉

**Pull request merged**